packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -25,14 +25,13 @@ import useInput from './use-input';
 import { store as blockEditorStore } from '../../store';
 
 export function useWritingFlow() {
-	const [ before, ref, after ] = useTabNav();
```

Comment:
Since block toolbar is no longer receiving focus via Shift+Tab, remove the before.